challenge-1/theme/assets/critical.css

```diff
@@ -5,3 +5,11 @@
 @import "./css/components/nav.css";
 
 @import "tailwindcss/utilities";
+
+/* Custom CSS */
+body{
+    font-family:Airbnb Cereal VF, Circular, -apple-system, BlinkMacSystemFont, Roboto,       Helvetica Neue, sans-serif
```

Comment:
this would probably be a good place to use tailwind `@apply` statements (as opposed to pure CSS)